test/playbacks/html5_video_spec.js

```diff
@@ -190,6 +190,14 @@ describe('HTML5Video playback', function() {
       expect(progress.current).to.be.equal(end[1])
     })
 
+    it('does not trigger PLAYBACK_BUFFERFULL when playback is initialized', function() {
+      let callback = sinon.spy()
+      this.playback.on(Events.PLAYBACK_BUFFERFULL, callback)
```

Comment:
Instead of using the created `playback` I'd create a new one here just to be sure that when it's initialized (aka instantiated, I guess) we're not facing this issue anymore.

```javascript
const playback = new HTML5Video(this.options)
```